Tools/MiniBrowser/playstation/MainWindow.cpp

```diff
@@ -106,9 +115,30 @@ MainWindow::MainWindow(const char* requestedURL)
     }});
 
     createNewWebView(nullptr);
+
+    // Parse command line options
+    String requestedURL = ""_s;
+
+    for (auto option : options) {
+        bool foundOption = false;
+        executeOption(option, &requestedURL);
+    }
+
     activeWebView()->loadURL(requestedURL ? requestedURL : "https://webkit.org");
 }
 
+void MainWindow::executeOption(const String& option, String* requestedURL)
+{
+    if (!strcasecmp(option, s_resourceLoadStatisticsEnabled)) {
```

Comment:
You should use `==` or `compare` here.

Reply:
Ah, you want to compare  strings case-insensitively. I don't think command switches should be case-insensitive. But, it's OK if you want.